lib/react_client/react_interop.dart

```diff
@@ -423,10 +425,26 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
 /// See [ReactDartInteropStatics], [createReactDartComponentClass].
 class ComponentStatics<T extends Component> {
   final ComponentFactory<T> componentFactory;
-
   ComponentStatics(this.componentFactory);
 }
 
+/// An object that stores static methods and information for a specific component class.
+///
+/// This object is made accessible to a component's JS ReactClass config, which
+/// passes it to certain methods in [ReactDartInteropStatics2].
+///
+/// [ComponentStatics2] exists in addition to [ComponentStatics] because
+/// [Component2] brings new static lifecycle methods that are exposted via
```

Comment:
Typo: "exposted"

Although, #nit I think this paragraph can just be removed; one can deduce from the absence of `instanceForStaticMethods` in `ComponentStatics` that this is the case